generators/client/generator.ts

```diff
@@ -108,8 +108,8 @@ export default class JHipsterClientGenerator extends BaseApplicationGenerator {
     return this.asComposingTaskGroup({
       async composing() {
         const { clientFramework, testFrameworks } = this.jhipsterConfigWithDefaults;
-        if ([ANGULAR, VUE, REACT].includes(clientFramework)) {
```

Comment:
Why? It doesn't impact types and is always useful for folding

Reply:
type of `[ANGULAR, VUE, REACT]` is `[string, string, string]`
type of `['angular', 'react', 'vue']` is `['angular', 'react', 'vue']`.
type of clientFramework is `'angular' | 'react' | 'vue'`

Reply:
It impacts types.